csharp/src/Ice/LocatorInfo.cs

```diff
@@ -90,83 +90,81 @@ internal void ClearCache(Reference reference)
         }
 
         /// <summary>Resolves an indirect reference using the locator proxy or cache.</summary>
-        internal async ValueTask<(EndpointList Endpoints, bool Cached)> ResolveIndirectReferenceAsync(
+        internal async ValueTask<(EndpointList Endpoints, TimeSpan EndpointsAge)> ResolveIndirectReferenceAsync(
             Reference reference,
+            TimeSpan endpointsMaxAge,
             CancellationToken cancel)
         {
             Debug.Assert(reference.IsIndirect);
 
             EndpointList endpoints = ImmutableArray<Endpoint>.Empty;
-            bool cached = false;
+            TimeSpan endpointsAge = TimeSpan.Zero;
 
             Location location = reference.Location;
-
             if (reference.IsWellKnown)
             {
                 // First, we check the cache.
-                (endpoints, location, cached) = GetResolvedWellKnownProxyFromCache(reference,
-                                                                                   reference.LocatorCacheTimeout);
-                if (!cached)
+                (endpoints, location, endpointsAge) = GetResolvedWellKnownProxyFromCache(reference);
```

Comment:
The "endpointsAge" variable name is not consistent with the naming of the return value field "LocationAge"

Reply:
Fixed